doc/topics/tutorials/lxc.rst

```diff
@@ -226,6 +226,17 @@ container-by-container basis, for instance using the ``nic_opts`` argument to
     for instance, typically are configured for eth0 to use DHCP, which will
     conflict with static IP addresses set at the container level.
 
+.. note::
+
+    For LXC < 1.0.7 and DHCP support, set ``ipv4.gateway: 'auto'`` is your
+    network profile, ie.::
+
+        lxc.network_profile.nic:
```

Comment:
@kiorky are you sure you mean lxc.network_profile.nic ? From the rest of the documentation I would think it might be either lxc.network_profile (new version) or lxc.nic (old version)